modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -321,7 +321,7 @@ export default class Ecdsa {
    * @returns {SignShareRT} Returns the participant private w-share
    * and k-share to be distributed to other participant signer
    */
-  async signShare(xShare: XShare | XShareWithNTilde, yShare: YShare | YShareWithNTilde): Promise<SignShareRT> {
+  async signShare(xShare: XShare | XShareWithNTilde, yShare: YShareWithNTilde): Promise<SignShareRT> {
```

Comment:
any reason to still have `XShare`? I think we can remove it since we're making it mandatory with range proofs.

Reply:
For this, you'll also need to make few more changes, see the diff
<img width="1044" alt="Screen Shot 2023-02-16 at 1 21 06 PM" src="https://user-images.githubusercontent.com/71043191/219375996-f683e04c-66f5-4c2f-a314-02c5046e919a.png">